src/pages/Calendar/Caption/caption.style.ts

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components/native";
+import { ColorScheme } from "@styles/globalStyles";
+import { FontScheme } from "@styles/globalStyles";
+
+export const Sc = {
+  Captions: styled.View`
+    flex-direction: row;
+    gap: 96px;
```

Comment:
Here the idea is that both sides shouldn't be touching, so instead of using gap, you could do the following, which allows the space to be dynamic instead of fixed

```js
  Captions: styled.View`
    flex-direction: row;
    justify-content: space-between;
    width: 100%;
  `,
```